config/initializers/session_store.rb

```diff
@@ -1,3 +1,2 @@
-# Be sure to restart your server when you modify this file.
+Champaign::Application.config.session_store :cookie_store
 
-Champaign::Application.config.session_store :redis_store, servers: 'redis://redis:6379/0/cache'
```

Comment:
`cookie_store` is fast, and less overhead for us.
http://api.rubyonrails.org/classes/ActionDispatch/Session/CookieStore.html